plugins/infinispan-snapshot/pom.xml

```diff
@@ -161,4 +161,33 @@
       </profile>
    </profiles>
 
+   <build>
+      <plugins>
+         <plugin>
+            <artifactId>maven-resources-plugin</artifactId>
+            <version>3.1.0</version>
+            <executions>
+               <execution>
```

Comment:
@alanfx we don't need to maint the resources in snapshot. it is dynamic now based on `${plugins.infinispan-latest}`